VoodooInput/VoodooInputSimulator/VoodooInputSimulatorDevice.cpp

```diff
@@ -195,6 +194,7 @@ void VoodooInputSimulatorDevice::constructReportGated(const VoodooInputEvent& mu
             finger_data.Size = 10;
             finger_data.Touch_Minor = 20;
             finger_data.Touch_Major = 20;
+            finger_data.Pressure = 10;
```

Comment:
Can you put Pressure assignment to the first line of the block? It will be easier to read this way due to similar code for supportsPressure block.